Interfaces.ts

```diff
@@ -0,0 +1,9 @@
+/**Interface to define the structure of grid data*/
```

Comment:
Please leave a space after the `/**` and before the `*/`. Also, I think this comment belongs on a different `interface`